### DISCUSSION TITLE
请问官方可以提供一个ubuntu部署cubefs集群的指导文档吗

### DISCUSSION THREAD

**Discussion by nyl1001** [cubefs/cubefs#3368] [General]
请问官方可以提供一个ubuntu部署cubefs集群的指导文档吗

**Comment by JasonHu520** [comment]
https://cubefs.io/zh/docs/master/quick-start/manual-deploy.html#%E9%9B%86%E7%BE%A4%E9%83%A8%E7%BD%B2 可以看看这个官网文档可以不？命令行的形式
